orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/DeployManifestContext.java

```diff
@@ -89,16 +89,19 @@ public TrafficManagement (
     }
 
     @Getter
-    static class Options {
+    public static class Options {
       private final boolean enableTraffic;
       private final List<String> services;
+      private final ManifestStrategyType strategy;
```

Comment:
I wonder if we can handle defaulting in the constructors here to avoid null checks in calling code.  I briefly thought of this when I was adding the new fields, but didn't implement it.  But if instead of defaulting to `Options` as `null`, we just made an `Options` with reasonable defaults (ex: strategy is none) then we could avoid some null checking.  Just an idea, not sure if it will be easier/harder to reason about.

Reply:
Oh, will the Options constructor not run at all if `Options` is null? I thought we were already defaulting the fields here: https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/DeployManifestContext.java#L101

Reply:
If the input JSON doesn't have `options` specified then it will just be null.  We could do something like:
```
this.options = Optional.ofNullable(options).orElse(new Options())
```

Reply:
My main goal here is just to see if we can reduce all the null-checks when we access this...but it may just move the complexity to here, so feel free to ignore this comment.  (I ignored my own comment this when I initially wrote this.)

Reply:
I have chosen not to ignore this comment :) Added defaults for `trafficManagement` and `options`, allowing us to remove the verbose check from `DeployManifestStage.afterStages`. Tested with a stage config with `trafficManagement: null` just to be sure and it works!